NumpySignaturesTest::testWrappedSignaturesMatch fails with Numpy 1.20

```
E     AssertionError: {'asarray': {'np_params': ['a', 'dtype', 'order', 'like'], 'jnp_params': ['a', 'dtype', 'order']}, 'corrcoef': {'np_params': ['x', 'y', 'rowvar', 'dtype'], 'jnp_params': ['x', 'y', 'rowvar']}, 'cov': {'np_params': ['m', 'y', 'rowvar', 'bias', 'ddof', 'fweights', 'aweights', 'dtype'], 'jnp_params': ['m', 'y', 'rowvar', 'bias', 'ddof', 'fweights', 'aweights']}, 'eye': {'np_params': ['N', 'M', 'k', 'dtype', 'like'], 'jnp_params': ['N', 'M', 'k', 'dtype']}, 'full': {'np_params': ['shape', 'fill_value', 'dtype', 'like'], 'jnp_params': ['shape', 'fill_value', 'dtype']}, 'identity': {'np_params': ['n', 'dtype', 'like'], 'jnp_params': ['n', 'dtype']}, 'ones': {'np_params': ['shape', 'dtype', 'like'], 'jnp_params': ['shape', 'dtype']}, 'tri': {'np_params': ['N', 'M', 'k', 'dtype', 'like'], 'jnp_params': ['N', 'M', 'k', 'dtype']}} != {}
E     Missing entries:
E     'asarray': {'np_params': ['a', 'dtype', 'order', 'like'], 'jnp_params': ['a', 'dtype', 'order']}
E     'corrcoef': {'np_params': ['x', 'y', 'rowvar', 'dtype'], 'jnp_params': ['x', 'y', 'rowvar']}
E     'cov': {'np_params': ['m', 'y', 'rowvar', 'bias', 'ddof', 'fweights', 'aweights', 'dtype'], 'jnp_params': ['m', 'y', 'rowvar', 'bias', 'ddof', 'fweights', 'aweights']}
E     'eye': {'np_params': ['N', 'M', 'k', 'dtype', 'like'], 'jnp_params': ['N', 'M', 'k', 'dtype']}
E     'full': {'np_params': ['shape', 'fill_value', 'dtype', 'like'], 'jnp_params': ['shape', 'fill_value', 'dtype']}
E     'identity': {'np_params': ['n', 'dtype', 'like'], 'jnp_params': ['n', 'dtype']}
E     'ones': {'np_params': ['shape', 'dtype', 'like'], 'jnp_params': ['shape', 'dtype']}
E     'tri': {'np_params': ['N', 'M', 'k', 'dtype', 'like'], 'jnp_params': ['N', 'M', 'k', 'dtype']}

tests/lax_numpy_test.py:5109: AssertionError
=========================================================================================== short test summary info ============================================================================================
FAILED tests/lax_numpy_test.py::NumpySignaturesTest::testWrappedSignaturesMatch - AssertionError: {'asarray': {'np_params': ['a', 'dtype', 'order', 'like'], 'jnp_params': ['a', 'dtype', 'order']}, 'corrcoe...
```

--- a/tests/lax_numpy_test.py
+++ b/tests/lax_numpy_test.py
@@ -5043,20 +5043,24 @@ class NumpySignaturesTest(jtu.JaxTestCase):
     # TODO(jakevdp): fix some of the following signatures. Some are due to wrong argument names.
     unsupported_params = {
       'angle': ['deg'],
+      'asarray': ['like'],
       'broadcast_to': ['subok', 'array'],
       'clip': ['kwargs'],
-      'corrcoef': ['ddof', 'bias'],
+      'corrcoef': ['ddof', 'bias', 'dtype'],
+      'cov': ['dtype'],
       'empty_like': ['subok', 'order'],
       'einsum': ['kwargs'],
       'einsum_path': ['einsum_call'],
-      'eye': ['order'],
-      'full': ['order'],
+      'eye': ['order', 'like'],
+      'identity': ['like'],
+      'full': ['order', 'like'],
       'full_like': ['subok', 'order'],
       'histogram': ['normed'],
       'histogram2d': ['normed'],
       'histogramdd': ['normed'],
-      'ones': ['order'],
+      'ones': ['order', 'like'],
       'ones_like': ['subok', 'order'],
+      'tri': ['like'],
       'zeros_like': ['subok', 'order']
     }
 

